core/trino-main/src/main/java/io/trino/server/ThrowableMapper.java

```diff
@@ -54,6 +55,11 @@ public Response toResponse(Throwable throwable)
 
         ResponseBuilder responseBuilder = Response.serverError()
                 .header(CONTENT_TYPE, TEXT_PLAIN);
+
+        if (!Strings.isNullOrEmpty(throwable.getMessage())) {
```

Comment:
static import, also I would check if string is not blank.

I would only add this header for Trino exception only is supposed to be have user facing message.